dev/tidb-in-kubernetes/maintain/lightning.md

```diff
@@ -0,0 +1,139 @@
+        ```
+
+* Configure an existing TiDB cluster to enable tikv-importer
+
+    1. Set `importer.create` to `true` in the `values.yaml` file of the TiDB cluster
+
+    2. Upgrade the existing TiDB cluster
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        helm upgrade <tidb-cluster-release-name> pingcap/tidb-cluster -f values.yaml --version=<chart-version>
+        ```
+
+## Deploy tidb-lightning
+
+1. Configure TiDB Lightning
+
+    Use the following command to get the default configuration of TiDB Lightning.
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm inspect values pingcap/tidb-lightning --version=<chart-version> > tidb-lightning-values.yaml
+    ```
+
+    TiDB Lightning Helm chart supports both local and remote data source.
+
+    * Local
+
+        Local mode requires the Mydumper backup data to be on one of the Kubernetes node. This mode can be enabled by setting `dataSource.local.nodeName` to the node name and `dataSource.local.hostPath` to the Mydumper backup data directory path which contains a file named `metadata`.
+
+    * Remote
+
+        Unlike local mode, remote mode needs to use [rclone](https://rclone.org) to download Mydumper backup tarball file from a network storage to a PV. Any cloud storage supported by rclone should work, but currently only the following have been tested: [Google Cloud Storage (GCS)](https://cloud.google.com/storage/), [AWS S3](https://aws.amazon.com/s3/), [Ceph Object Storage](https://ceph.com/ceph-storage/object-storage/).
+
+        1. Make sure that `dataSource.local.nodeName` and `dataSource.local.hostPath` are commented out.
+
+        2. Create a `Secret` containing the rclone configuration. A sample configuration is listed below. Only one cloud storage configuration is required. For other cloud storages, please refer to [rclone documentation](https://rclone.org/).
+
+            {{< copyable "" >}}
+
+            ```yaml
+            apiVersion: v1
+            kind: Secret
+            metadata:
+              name: cloud-storage-secret
+            type: Opaque
+            stringData:
+              rclone.conf: |
+              [s3]
+              type = s3
+              provider = AWS
+              env_auth = false
+              access_key_id = <my-access-key>
+              secret_access_key = <my-secret-key>
+              region = us-east-1
+
+              [ceph]
+              type = s3
+              provider = Ceph
+              env_auth = false
+              access_key_id = <my-access-key>
+              secret_access_key = <my-secret-key>
+              endpoint = <ceph-object-store-endpoint>
+              region = :default-placement
+
+              [gcs]
+              type = google cloud storage
+              # The service account must include Storage Object Viewer role
+              # The content can be retrieved by `cat <service-account-file.json> | jq -c .`
```

Comment:
```suggestion
              # The content can be retrieved by `cat <service-account-file.json> | jq -c`.
```
Just to doublecheck, do you mean `cat <service-account-file.json> | jq -c` here?